typescript/lambda-provisioned-concurrency-autoscaling/lib/lambda-pc-autoscaling-schedule.ts

```diff
@@ -0,0 +1,59 @@
+import { Duration, Stack, StackProps} from "aws-cdk-lib";
+import { Function, Runtime } from "aws-cdk-lib/aws-lambda";
+import { Construct } from 'constructs';
+
+import { ScalableTarget, Schedule, ServiceNamespace } from 'aws-cdk-lib/aws-applicationautoscaling';
+import { Metric }from 'aws-cdk-lib/aws-cloudwatch';
+import { NodejsFunction } from 'aws-cdk-lib/aws-lambda-nodejs';
+import { Alias } from 'aws-cdk-lib/aws-lambda';
+
+
+
+interface LambdaPCScalingScheduleStackProps extends StackProps {
+  functionName: string
+}
+
+export class LambdaPCScalingScheduleStack extends Stack {
+  private lambdaFunction: Function
+  private lambdaFunctionName: string
+
+  constructor(scope: Construct, id: string, props: LambdaPCScalingScheduleStackProps) {
+    super(scope, id, props);
+    this.lambdaFunctionName = props.functionName
+
+    // Create Sample Lambda Function which will create metrics
+    this.lambdaFunction = new NodejsFunction(this, 'LambdaFunction', {
+      functionName: this.lambdaFunctionName,
+      entry: `./lambda/lambda-handler.ts`,
+      runtime: Runtime.NODEJS_16_X,
```

Comment:
Please update to 18_X.  Check issue #961 for more details on why. Lambda is simple to so it should not need more changes.

Reply:
Thanks @kaizio, update done